tests/functional_small/ops/ops_eigen_level2.cc

```diff
@@ -7,25 +7,39 @@
   M_t M(3,3);                  \
   M << 1.,0.,2.,2.,1.,3.,0.,0.,1.;      \
   Eigen::VectorXd myR(3);      \
+  myR(0) = NAN; /* simulate uninitialized Nan */ \
```

Comment:
what is this NAN? 
can you please use C++ https://en.cppreference.com/w/cpp/numeric/math/nan?

Reply:
Thank you for pointing that out! Fixed on https://github.com/Pressio/pressio/pull/354/commits/ea1b72890d3fd746c46ae60750133c16595d2568.